elements/pl-code/pl-code.py

```diff
@@ -70,18 +55,98 @@
 ]
 
 
+class NoHighlightingLexer(pygments.lexer.Lexer):
+    def __init__(self, **options):
+        pygments.lexer.Lexer.__init__(self, **options)
+        self.compress = options.get('compress', '')
+        self.name = "No Highlighting Lexer"
+        self.aliases = ["none"]
+        self.filenames = ["*.none"]
+        self.mimetypes = []
+
+    def get_tokens_unprocessed(self, text):
+        return [(0, Token.Text, text)]
+
+
+class HljsFormatter(pygments.formatter.Formatter):
```

Comment:
I'm not very familiar with how Pygments works, but is this class so that we can emulate the look of Highlight.js using Pygments? Does Pygments have a default style that we could use instead? I don't think there is any particular need to maintain backwards compatibility with the previous highlighting?

Reply:
It does have a default style, I don't think it looks particularly great but it's something.

Reply:
Ok, I'll leave it up to you. I'm generally in favor of using defaults and not having our own special code unless there's really a significant gain. I haven't seen the Pygments default style so I don't know how bad it is 😄

Reply:
Sure, I think it actually may come with a bunch of styles built-in so I'll see if any of those look nice

Reply:
Awesome, if any of them are acceptable then that will let us have less code to maintain in the future 😄